src/mmw/apps/core/mail/backends/boto_ses_mailer.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import json
+import logging
+import sys
+import traceback
+import boto.ses
+
+from django.conf import settings
+from django.core.mail.backends.base import BaseEmailBackend
+
+from django_statsd.clients import statsd
+from email.mime.text import MIMEText
+
+logger = logging.getLogger(__name__)
+
+
+class EmailBackend(BaseEmailBackend):
+
+    def __init__(self, fail_silently=False, aws_region='us-east-1', *args,
+                 **kwargs):
+        super(EmailBackend, self).__init__(fail_silently=fail_silently)
+        self.mailer = BotoMailer(aws_region)
```

Comment:
Since we're already depending on [instance metadata](http://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/ec2-instance-metadata.html#instancedata-data-categories) for authentication credentials, we can also pull the region out by trimming the availability zone.